src/legacy/js/functions/_t16ReleaseEditor.js

```diff
@@ -87,36 +87,24 @@ function releaseEditor(collectionId, data) {
     });
 
     /* The checked attribute is a boolean attribute, which means the corresponding property is true if the attribute is present at all—even if, for example, the attribute has no value or is set to empty string value or even "false" */
-    var checkBoxStatus = function (id) {
-        if (id === 'natStat') {
-            if (data.description.nationalStatistic === "false" || data.description.nationalStatistic === false) {
-                return false;
-            } else {
-                return true;
-            }
-        } else if (id === 'cancelled') {
-            if (data.description.cancelled === "false" || data.description.cancelled === false) {
-                return false;
-            } else {
-                return true;
-            }
-        } else if (id === 'finalised') {
-            if (data.description.finalised === "false" || data.description.finalised === false) {
-                return false;
-            } else {
-                return true;
-            }
+    var checkBoxStatus = function (value) {
```

Comment:
The comment above this line does not seem to agree with what this function is doing. I might be misunderstanding it but it seems to suggest that if the value is "" or "false" then it should return true?

Reply:
Yes, you are right! The comment does not seem to make much sense. It is present in several files and I have copied it over without thinking about it too much. I will amend them!